src/js/module.js

```diff
@@ -262,6 +269,35 @@ iotjs_module_t.load = function(id, parent) {
 };
 
 
+function _makeRequireFunction(mod) {
+  var Module = mod.constructor;
+  function require(id) {
+    return mod.require(id);
+  }
+
+  function _resolve(request) {
+    if (!util.isString(request)) {
+      throw new TypeError('module must be string');
+    }
+
+    if (process.builtin_modules[request]) {
+      return request;
+    }
+
+    var path = Module.resolveModPath(request, mod);
+    if (!path) {
+      throw new Error('Module not found: ' + request);
+    }
+    return path;
+  }
+  require.resolve = _resolve;
+  require.main = mainModule;
+  require.cache = Module._cache;
```

Comment:
There is no property `_cache` on `iotjs_module_t`

Reply:
Yeah, It is Module.cache; I am sorry!

Reply:
There is nothing to be sorry 😄